metadata.rb

```diff
@@ -6,8 +6,7 @@
 chef_version     '>= 14'
 license          'Apache-2.0'
 description      'Installs and configures nginx'
-long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '10.0.1'
+version          '10.0.2'
```

Comment:
Hey, 
Please don't bump version numbers as we prefer to manage these ourselves 

It is common for us to bundle up releases together etc :)